InRule.Catalog.Service.json

```diff
@@ -85,7 +85,7 @@
     "catalogSqlDbPerformanceLevel": {
       "type": "string",
       "defaultValue": "Basic",
-      "allowedValues": [ "Basic", "S0", "S1", "S2", "S3", "P1", "P2", "P4", "P6", "P11", "P15" ],
+      "allowedValues": [ "Basic", "S0", "S1", "S2", "S3", "S4", "S6", "S7", "S9", "S12", "P1", "P2", "P4", "P6", "P11", "P15" ],
```

Comment:
Are these represented in the portal UI?

Reply:
Yes, that's where I checked to see if we needed to update this.